app/models/edition/related_policies.rb

```diff
@@ -1,6 +1,16 @@
 module Edition::RelatedPolicies
   extend ActiveSupport::Concern
 
+  def delete_policy(content_id)
+    edition_policies.delete(EditionPolicy.where(policy_content_id: content_id))
```

Comment:
In which case, maybe `edition_policies.where(policy_content_id: content_id).delete_all`? Although in terms of queries executed I think they're equivalent.